packages/tests/src/utils/azureCliHelper.ts

```diff
@@ -48,8 +48,7 @@ export class AzSqlHelper {
   public async createTable(sqlServerEndpoint: string) {
     // login
     console.log(`Logging in...`);
-    const { success: loginSuccess } = await AzSqlHelper.login();
-    if (!loginSuccess) return;
```

Comment:
why need remove this line?

Reply:
![image](https://github.com/OfficeDev/TeamsFx/assets/67082457/4dac74a1-74b2-4008-81cf-4a1599a04da1)
this warning message will block the success result.